nsv13/code/modules/power/stormdrive.dm

```diff
@@ -1662,6 +1684,100 @@ Control Rods
 	explosion(src, 0, 0, 1, 18, FALSE)
 	do_sparks(10, FALSE, src)
 
+/obj/effect/anomaly/stormdrive/squall //Gravity + Brute
+	name = "storm squall anomaly"
+	icon_state = "dragnetfield"
+	alpha = 65 //Not exactly easy to see
+
+/obj/effect/anomaly/stormdrive/squall/anomalyEffect()
+	var/direction = pick(GLOB.alldirs)
+	var/times = rand(2, 4)
+	for(var/I = 0, I < times, I++) //Appears to hop around, but actually passes through mobs
+		step(src, direction)
+
+	if(prob(90))
+		var/turf/open/L = get_turf(src)
+		if(!istype(L) || !L.air)
+			return
+		var/datum/gas_mixture/env = L.return_air()
+		var/temperature = env.return_temperature()
+		atmos_spawn_air("o2=5,n2=15;TEMP=[temperature]") //Add a little extra oxygenated air
+
+	else //throw everything away
+		for(var/mob/living/M in orange(4, src))
+			M.Paralyze(20)
+			M.Dizzy(5)
+			M.apply_damage(10)
+			var/atom/target = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
+			M.throw_at(target, 4, 2)
+		for(var/obj/O in orange(4, src))
+			if(!O.anchored)
+				var/atom/target = get_edge_target_turf(O, get_dir(src, get_step_away(O, src)))
+				O.throw_at(target, 4, 2)
+
+/obj/effect/anomaly/stormdrive/squall/Crossed(mob/living/M)
+	polarise(M)
+
+/obj/effect/anomaly/stormdrive/squall/Bump(mob/living/M)
+	polarise(M)
+
+/obj/effect/anomaly/stormdrive/squall/Bumped(atom/movable/AM)
+	polarise(AM)
+
```

Comment:
And we don't need to send comsigs or call the default procs?

Reply:
This is just how anomalies work?

Reply:
scream